### PR TITLE
feat: add builder-projects-panel scenes general menu section

### PR DESCRIPTION
**Pull request by pbosio**
Splitting feature Builder Projects Panel in several PR to avoid a huge PR:
* add Builder Projects Panel scenes general section

### CODE REVIEW

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/MenuSections/SectionScenesController.cs [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    internal Dictionary<string, SceneCardView> projectViews;
+    private List<SceneSearchInfo> searchList = new List<SceneSearchInfo>();
+
+    public SectionScenesController()
+    {
+        var prefab = Resources.Load<SectionScenesView>("BuilderProjectsPanelMenuSections/SectionScenesView");
+        view = Object.Instantiate(prefab);
+
+        view.btnProjectsViewAll.onClick.AddListener(()=> RequestOpenSection(SectionsController.SectionId.SCENES_PROJECT));
+        view.btnInWorldViewAll.onClick.AddListener(()=> RequestOpenSection(SectionsController.SectionId.SCENES_DEPLOYED));
```
Comment (sandrade-dcl) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/MenuSections/SectionScenesController.cs:34]: Shouldn't we unsubscribe these 2 subscriptions in some place (maybe in the `Dispose` function)?

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/SectionsController.cs [hunk 2]
```diff
@@ -41,6 +43,9 @@ public SectionsController(ISectionFactory sectionFactory, Transform sectionsPare
     {
         this.sectionsParent = sectionsParent;
         this.sectionFactory = sectionFactory;
+
+        SectionBase.OnRequestOpenSection += (sectionId) => OnRequestOpenSection?.Invoke(sectionId);
+        SectionBase.OnRequestContextMenuHide += ()=> OnRequestContextMenuHide?.Invoke();
```
Comment (sandrade-dcl) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/SectionsController.cs]: Same as before.

**Review by sandrade-dcl — CHANGES_REQUESTED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/MenuSections/SectionScenesController.cs (hunk 1 above)
Comment (pbosio) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/MenuSections/SectionScenesController.cs:34]: Dispose will Destroy view.gameObject and those two buttons are inside that GameObject so there is really no need to unsubscribe to those click events

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/SectionsController.cs (hunk 2 above)
Comment (pbosio) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/SectionsController.cs]: Yeap, I must unsubscribe those on Dispose :) thx!

**Review by sandrade-dcl — APPROVED** [review]

**Review by BrianAmadori — APPROVED** [review]
The styling on this PR is top notch. Loving the use of interfaces, search handler and lean unit tests. I'd have to get deep into this to find anything useful to comment. Congratulations!. 🏆

**Pull request merged**